hpecp/client.py

```diff
@@ -47,6 +47,7 @@
 from .logger import Logger
 from .tenant import TenantController
 from .catalog import CatalogController
+from .role import RoleController
```

Comment:
If you are using vscode (or git pod), use the 'Organiza imports' or 'Sort imports' option (Ctrl + Shift + P and then choose the appropriate command) to sort the imports as per PEP.

Reply:
Got it!
Thanks a ton for the help and review :")